stable/opsportal/values.yaml

```diff
@@ -120,8 +120,18 @@ kommander:
   kubefed:
     enabled: false
 
+  # disable kommander-cluster-lifecycle
+  kommander-cluster-lifecycle:
```

Comment:
Excuse my ignorance, but I'm confused why opsportal has any references to kommander or kubefed?

Reply:
Your ignorance is excused :). Kommander is pulls as a subchart, as such, if I do not place these, opsportal will deploy all these extra tools meant for kommander.